include/x/cba/addons/main/script_macros_common.hpp

```diff
@@ -478,8 +478,8 @@ Group: General
 
 // *************************************
 // Internal Functions
-#define DOUBLES(var1,var2) var1##_##var2
-#define TRIPLES(var1,var2,var3) var1##_##var2##_##var3
+#define DOUBLES(var1,var2) ##var1##_##var2
```

Comment:
This was changed in ACE3 (and branch on CBA) for armake, as leading concatenation is pointless.

Reply:
Should get these fixes merged into CBA, instead of waiting for full armake merge on CBA.

Reply:
Yes, indeed.